services/victoriametrics/scrape_configs.go

```diff
@@ -381,6 +385,18 @@ func scrapeConfigsForMongoDBExporter(s *models.MetricsResolutions, params *scrap
 	if hr != nil {
 		r = append(r, hr)
 	}
+	lr, err := scrapeConfigForStandardExporter("lr", s.LR, params, []string{
+		"dbstats",
+		"indexstats",
+		"collstats",
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	if lr != nil {
+		r = append(r, lr)
+	}
```

Comment:
Better to check for emptiness, not for nil. But in general, you can append without this check at all.

Reply:
we are checking object, not array.

Reply:
Ah, got it 👌
I watched the wrong function.